main.py

```diff
@@ -246,7 +236,8 @@ def escape_path(path):
 		short_anim_arg = ',short_animation' if settings.settings['short_anim'] else ''
 		hdd_lock_arg = ',locked' if settings.settings['hdd_locked'] else ''
 		sys_memory = settings.settings['sys_memory'].split(' ')[0]+'M'
-		accelerator_arg = Xqemu.generateAcceleratorArg(settings.settings['use_accelerator'])
+		kvm_irqchip_arg = ',kernel_irqchip=off' if platform.system == 'Linux' and settings.settings['use_accelerator'] else ''
```

Comment:
nit: this is a bit ugly (2 checks on one line / ternary operator).

Why not:
```python
if settings.settings['use_accelerator']:
    kvm_irqchip_arg = ',kernel_irqchip=off' if platform.system == 'Linux' and else ''
    accel_arg = ',accel=kvm:whpx:hax:tcg%(kvm_irqchip_arg)s' % locals()
else:
    accel_arg = ''
```

or, if you want to keep the current style:
```python
if settings.settings['use_accelerator']:
    kvm_irqchip_arg = ',kernel_irqchip=off' if platform.system == 'Linux' else ''
accel_arg = ',accel=kvm:whpx:hax:tcg%(kvm_irqchip_arg)s' % locals() if settings.settings['use_accelerator'] else ''
```
*(You could also drop the condition and accept that it might be unused)*

Which relies on the fact, that the LHS is never evaluated if the condition is false (and `kvm_irqchip_arg` isn't set)

Reply:
Looking at this again, we should just always use: `,accel=kvm:whpx:hax:tcg,kernel_irqchip=off` unless that causes an error on some platform.

No need for unnecessary complexity.